src/soc/rockchip/common.h

```diff
@@ -0,0 +1,67 @@
+#ifndef COMMON_H
```

Comment:
```suggestion
#ifndef __WIRINGX_SOC_ROCKCHIP_COMMON_H__
```
万一你这个文件被别人引用了，然后他也有个common.h呢？